.gitignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
use `Prettier` to avoid these formatting issues

Reply:
I have added prettier.